backend/battle/forms.py

```diff
@@ -90,11 +109,11 @@ def save(self, commit=True):
         data = self.clean()
```

Comment:
`data = self.clean()` -> `cleaned_data = self.clean()`

Because forms have both data and cleaned_data, so this will avoid confusion.

Reply:
I don't understand, should I use data instead of cleaned_data?

Reply:
just the variable name, call it cleaned_data

Reply:
Sure, I'll change this.